requirements.txt

```diff
@@ -3,4 +3,5 @@ numpy>=1.17
 openqasm3[parser]>=0.4.0
 qbraid-core>=0.1.25
 pydantic>2.0.0
-typing-extensions>=4.0.0
\ No newline at end of file
+typing-extensions>=4.0.0
+pyqasm>=0.1.0
```

Comment:
Maybe change to prelease e.g. `pyqasm>=0.1.0a0` so we can run our integration tests?